src/loader.js

```diff
@@ -348,6 +348,9 @@
                             });
                             break;
                         default:
+                            if(parseInt(state.__version.split('.')[1]) === 14){
```

Comment:
Needs to check the major version as well:
In the future we will have a 2.14.x

Reply:
I considered this but thought that this code likely wouldn't exist in 2.0+